spyder/app/tests/test_mainwindow.py

```diff
@@ -72,13 +67,6 @@
 # =============================================================================
 # ---- Constants
 # =============================================================================
-# Location of this file
-LOCATION = osp.realpath(osp.join(os.getcwd(), osp.dirname(__file__)))
-
-# Time to wait until the IPython console is ready to receive input
-# (in milliseconds)
-SHELL_TIMEOUT = 40000 if os.name == 'nt' else 20000
-
 # Need longer EVAL_TIMEOUT, because need to cythonize and C compile ".pyx" file
 # before import and eval it
 COMPILE_AND_EVAL_TIMEOUT = 30000
```

Comment:
Should this and other constants and utility functions below be moved also to the `conftest.py` file ?

Reply:
Good idea. I'll push a commit moving everything that it's not a test to `conftest.py`.